src/qt/pivx/settings/settingsdisplayoptionswidget.cpp

```diff
@@ -101,9 +101,9 @@ SettingsDisplayOptionsWidget::SettingsDisplayOptionsWidget(PIVXGUI* _window, QWi
     setCssBtnSecondary(ui->pushButtonClean);
 
     initLanguages();
-    connect(ui->pushButtonSave, SIGNAL(clicked()), parent, SLOT(onSaveOptionsClicked()));
-    connect(ui->pushButtonReset, SIGNAL(clicked()), this, SLOT(onResetClicked()));
-    connect(ui->pushButtonClean, SIGNAL(clicked()), parent, SLOT(onDiscardChanges()));
+    connect(ui->pushButtonSave, &QPushButton::clicked, [=] { Q_EMIT saveSettings(); });
```

Comment:
Lambda `[=]` will capture all of the outer scope variables by value. Use `[this]` instead to capture the object by reference.

(Same in the signal connect that it's two lines below this one).

Reply:
done